examples/tensorflow-comparison/comparison-training-multigpu.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
This should have words, and titles, and comments. It shouldn't just be a notebook of only code.